spec/request/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'Delete Targets', type: :request do
```

Comment:
You could just remove this line and put the lines below inside the other describe.